style.css

```diff
@@ -149,6 +158,10 @@ a:hover {text-shadow: 0.05vw 0.05vw 0.05vw #527ECC;} /* mouse over link */
 		text-align: center;
 	}
 	
+	.navbar-links li2{
```

Comment:
#allows the seperator on my navbar to disappear when the screen is width is decreased and the burger menu is used